plugin/src/main/java/org/opensearch/ml/task/MLTaskRunner.java

```diff
@@ -129,4 +128,11 @@ public void dispatchTask(
     protected abstract TransportResponseHandler<Response> getResponseHandler(ActionListener<Response> listener);
 
     protected abstract void executeTask(Request request, ActionListener<Response> listener);
+
+    protected void checkCBAndExecute(FunctionName functionName, Request request, ActionListener<Response> listener) {
+        if (functionName != FunctionName.REMOTE) {
```

Comment:
So this PR will ignore circuit breaker checking for remote model. Can you add this to the description part ?

Reply:
added